### PR TITLE
fix: bump required google provider to 5.6 for postgres

### PR DESCRIPTION
**Pull request by ykyr**
Fixes https://github.com/terraform-google-modules/terraform-google-sql-db/issues/552

### CODE REVIEW

File: modules/postgresql/versions.tf [hunk 1]
```diff
@@ -27,11 +27,11 @@ terraform {
     }
     google = {
       source  = "hashicorp/google"
-      version = ">= 4.80.0, < 6"
+      version = ">= 5.6.0, < 6"
```
Comment (imrannayer) [inline modules/postgresql/versions.tf:30]: can u pl change it to

```
version = ">= 5.6, < 6"
```

Comment (ykyr) [reply modules/postgresql/versions.tf]: ah, right. Fixed. 
Thanks!

**Review by imrannayer — APPROVED** [review]

**Comment by apeabody** [comment]
Looks like this should be marked as a breaking change?

**Pull request merged**